### DISCUSSION TITLE
IDEA 怎么使用得意黑？

### DISCUSSION THREAD

**Discussion by luuuzh** [atelier-anchor/smiley-sans#91] [Q&A]
在设置中找不到咱这个字体

**Comment by yvvt0379** [comment]
你得先安装上去才能用啊。。。
PS：IDE不推荐使用这样的艺术体，还是老老实实用等宽字体吧

**Comment by ZSCMRMZ** [comment]
不推荐ide用美术字，很影响可读性，等宽最好

**Comment by Steven-Zhl** [comment]
我曾在PyCharm中使用过这个字体，安装后字体名为它的英文名称Smiley Sans。但确实如前面两位所说，视觉效果并不好，且不支持连写，不适合作为代码字体。
